yesod-core/yesod-core.cabal

```diff
@@ -1,5 +1,5 @@
 name:            yesod-core
-version:         1.6.11
+version:         1.6.11.1
```

Comment:
This should be a minor version bump, to 1.6.12, since it does affect behavior.

Reply:
Fixed :+1: and rebased to fix the commit message.